### ISSUE TITLE
allow no due date

### ISSUE THREAD

**Issue by radix** [posguy99/todoist#1]
It seems that newer versions of this script always set a due date, without the option to leave one off. It _seems_ it should be possible according to the Todoist API: https://todoist.com/API/#Passing%20in%20date_string%20and%20due_date

**Comment by posguy99** [comment]
> On Apr 3, 2015, at 10:33 AM, Christopher Armstrong notifications@github.com wrote:
> 
> It seems that newer versions of this script always set a due date, without the option to leave one off. It seems it should be possible according to the Todoist API: https://todoist.com/API/#Passing%20in%20date_string%20and%20due_date https://todoist.com/API/#Passing%20in%20date_string%20and%20due_date
> —
> Reply to this email directly or view it on GitHub https://github.com/posguy99/todoist/issues/1.

Yeah, I changed that when I put the defaults setting in to control the due date.  It is completely possible.  Maybe what I should do is set it up such that if the defaults entry isn’t there, it doesn’t apply one, versus applying a canned one.

## 

Marc Wilson
msw@cox.net mailto:msw@cox.net
posguy99@gmail.com mailto:posguy99@gmail.com
